test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -37,22 +38,17 @@ Describe "Test-ModuleManifest tests" -tags "CI" {
             "ModuleList"="Modules_InvalidModuleListinModuleManifest";
             "TypesToProcess"="Modules_InvalidManifest";
             "FormatsToProcess"="Modules_InvalidManifest";
-            "RootModule"="Modules_InvalidManifest";
+            #"RootModule"="Modules_InvalidManifest";
```

Comment:
Should be uncommented.  Looks like the cmdlets doesn't do any validation of RootModule currently, will need to add that code.